.github/workflows/xtensa.yml

```diff
@@ -55,22 +55,36 @@ jobs:
           echo ${{ secrets.TFLM_BOT_PACKAGE_READ_TOKEN }} | docker login ghcr.io -u tflm-bot --password-stdin
           docker run --rm -v `pwd`:/opt/tflite-micro ghcr.io/tflm-bot/xtensa:0.1 /opt/tflite-micro/tensorflow/lite/micro/tools/ci_build/test_xtensa_hifi5.sh
 
+  vision_p6_presubmit:
+    runs-on: ubuntu-latest
+
+    if: |
+      (github.event_name == 'pull_request_target' && contains(github.event.pull_request.labels.*.name, 'ci:run'))
```

Comment:
nit: Would github.event_name == 'workflow_dispatch' still be useful here?  just in case someone trying to manually trigger workflow for xtensa only?

Reply:
done